model/src/test/java/com/basistech/rosette/dm/AnnotatedTextTest.java

```diff
@@ -464,4 +492,34 @@ public void nullToData() {
         AnnotatedText text = new AnnotatedText.Builder().build();
         assertNull(text.toString());
     }
+
+    @Test
+    public void multipleEntriesWithSameKeyCrash() throws Exception {
```

Comment:
Should this test be in `CompatibilityV100ToV110Test`?  Also, feel free
to select a better name now that you know what the issue was.